src/epsimage.cpp

```diff
@@ -105,6 +105,11 @@ namespace {
     // closing part of all valid XMP trailers
     const std::string xmpTrailerEnd = "?>";
 
+    bool startsWith(const std::string& s, const std::string& start)
```

Comment:
You could replace the parameters with std::string_view. This would also allow making it constezpr and reduce binary size.

Reply:
Totally right. I still do not have the habit of using string_view. Thanks to having reviews from you, I hope to gain that habit 😉

Reply:
Note that constexpr also implies inline. In Gerbera, I've replaced all .find() == 0 usages with startswith. The function is done a bit differently though: https://github.com/gerbera/gerbera/blob/master/src/util/tools.h#L62